en/config/README.md

```diff
@@ -1,53 +1,77 @@
-:::
 
-The sub topics cover each of the steps in detail (first install the PX4 firmware and set your vehicle airframe):
-* [Firmware](../config/firmware.md)
-* [Airframe](../config/airframe.md)
-* [Sensor Orientation](../config/flight_controller_orientation.md)
-* [Compass](../config/compass.md)
-* [Gyroscope](../config/gyroscope.md)
-* [Accelerometer](../config/accelerometer.md)
-* [Airspeed](../config/airspeed.md) (primarily for fixed-wing/VTOL)
-* [Level Horizon Calibration](../config/level_horizon_calibration.md)
-* [Radio Setup](../config/radio.md)
-* [Joystick Setup](../config/joystick.md)
-* [Flight Modes](../config/flight_mode.md) (Optional)
-* [Battery](../config/battery.md) (optional)
-* [Safety](../config/safety.md) (optional)
-* [Actuators](../config/actuators.md)
-* [Autotune](../config/autotune.md)
+## Setup
 
+The following topics cover each of the steps in detail.
+You must first load firmware and select your vehicle airframe.
+Most other steps can be done out of order, except for tuning, which must be done last.
 
-## Video Guide
+Firmware/Vehicle selection (do this first):
 
-The video below shows the calibration process in detail (this uses an older version of *QGroundControl*, but most of the process is unchanged).
+- [Loading Firmware](../config/firmware.md)
+- [Vehicle (Frame) Selection](../config/airframe.md)
 
-@[youtube](https://youtu.be/91VGmdSlbo4)
+Motor/actuator setup:
+
+- [ESC Calibration](../advanced_config/esc_calibration.md)
+- [Actuator Configuration and Testing](../config/actuators.md)
+
+Sensor calibration:
+
+- [Sensor Orientation](../config/flight_controller_orientation.md)
+- [Compass Calibration](../config/compass.md)
+- [Gyroscope Calibration](../config/gyroscope.md)
+- [Accelerometer Calibration](../config/accelerometer.md)
+- [Airspeed Calibration](../config/airspeed.md) (Fixed-wing/VTOL only)
+- [Level Horizon Calibration](../config/level_horizon_calibration.md)
 
+Manual control setup (optional):
 
-## Advanced Configuration
+- [Radio Controller (RC) Setup](../config/radio.md)
+- [Joystick Setup](../config/joystick.md)
+- [Flight Mode Configuration](../config/flight_mode.md)
 
-If your [selected airframe configuration](../config/airframe.md) is a _specific vehicle model_ (e.g. [Holybro s500](../frames_multicopter/holybro_s500_v2_pixhawk4.md)) then it may benefit from fine tuning, but this is not generally required.
+Safety configuration:
 
-If you're using a new airframe, a "generic" airframe, significantly modifying an airframe, or using less common sensors then see:
-- [Advanced Configuration](../advanced_config/README.md): Vehicle-specific tuning, fine tuning, factory-level configuration.
-- [Flight Controller Peripherals](../peripherals/README.md) - Hardware and software setup related to specific hardware (in particular lesser-used sensors).
+- [Battery and Power Module Setup](../config/battery.md) (optional)

```

Comment:
FYI, I put battery under safety, because to some extent that is what it is. Happy for suggestions of another name that captures "general important feature config"